curriculum/challenges/english/14-responsive-web-design-22/learn-responsive-web-design-by-building-a-piano/612ec19d5268da7074941f84.md

```diff
@@ -6,8 +6,17 @@ dashedName: step-32
 ---
 
 # --description--
+In order to query two media conditions, the `and` logical operator is used. This one of many used to construct media queries.
```

Comment:
I think this is a bit more straightforward language:
```suggestion
Logical operators can be used to construct more complex media queries. The `and` logical operator is used to query two media conditions.
```

Reply:
I have edited Jeremy comment for the needed format, and to avoid spamming the pr.